index.js

```diff
@@ -447,6 +430,26 @@ function commandHandler(fullCommand, sendFunction, topic) { // If a status is se
                         console.log('Error in changeDate :', err);
                     });
                     break;
+                case 'resumerecord':
```

Comment:
IMO, if there is no argument in resumerecord/pauserecord, it should start/stop every type of measurements